### PR TITLE
unless the user is using registration.py directly, no longer log credentials

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

File: src/deployment/deploy.py [hunk 1]
```diff
@@ -894,23 +886,27 @@ def update_registration(self) -> None:
...
-        logger.info(
-            "Update your CLI config via: onefuzz config --endpoint "
-            "https://%s.azurewebsites.net --authority %s --client_id %s %s %s",
-            self.application_name,
-            self.cli_config["authority"],
-            self.cli_config["client_id"],
-            client_secret_arg,
-            multi_tenant_domain,
-        )
+            cmd += ["--tenant_domain", str(self.multi_tenant_domain)]
```
Comment (ranweiler) [inline src/deployment/deploy.py:905]: Not new, but this should be `"--multi_tenant_domain"`.

File: src/deployment/registration.py [hunk 2]
```diff
@@ -402,6 +402,8 @@ def create_and_display_registration(
     registration_name: str,
     approle: OnefuzzAppRole,
     subscription_id: str,
+    *,
+    display_secret: bool = False,
```
Comment (ranweiler) [inline src/deployment/registration.py:406]: We should document this.

Comment (ranweiler) [reply src/deployment/registration.py:406]: And consider giving it a scarier name.

**Review by ranweiler — APPROVED** [review]

File: src/deployment/deploy.py (hunk 1 above)
Comment (bmc-msft) [reply src/deployment/deploy.py:905]: no, it shouldn't.  The CLI doesn't take a `multi_tenant_domain`.   The CLI takes `tenant_domain`.

**Pull request merged**